Angular/Angular guidelines and best practices/Formatting, naming and best practices.md

```diff
@@ -586,6 +586,23 @@ const observable$: Observable;
 const mySubject$: Subject;
 ```
 
+The `$` suffix is here to indicate and let you know it is something you can subscribe to, but it does not let you know if you can change the observable in case it some type of a `Subject`, ie. `ReplaySubject` or `BehaviorSubject`. This naming convention goes along with the `no-exposed-subjects` RxJS ESLint rule. The aforementioned rule will cause an error in case you have publicly exposed` Subject`. The reasoning behind this is that subjects should never be publicly exposed unless converted to an observable using some pipeable operators or for an example, `asObservable` function.
```

Comment:
```suggestion
The `$` suffix is here to indicate and let you know it is something you can subscribe to, but it does not let you know if you can change the observable in case it is some type of a `Subject`, ie. `ReplaySubject` or `BehaviorSubject`. This naming convention goes along with the `no-exposed-subjects` RxJS ESLint rule. The aforementioned rule will cause an error in case you have publicly exposed` Subject`. The reasoning behind this is that subjects should never be publicly exposed unless converted to an observable using some pipeable operators or for example, `asObservable` function.
```